pandas/core/arrays/base.py

```diff
@@ -446,13 +446,24 @@ def shift(self, periods=1):
         Returns
         -------
         shifted : ExtensionArray
+
+        Notes
+        -----
+        If ``self`` is empty or ``periods`` is 0, a copy of ``self`` is
+        returned.
+
+        If ``periods`` > ``len(self)``, then an ExtensionArray of size
```

Comment:
you can remove the inner set of backticks, so this is all in one.  \`\`periods > len(self)\`\`.

Maybe change ExtensionArray to just "array", since we don't know the type of the concrete subclass.